### PR TITLE
Add rjweb-server Example

### PR DESCRIPTION
**Pull request by rotvproHD**

### CODE REVIEW

File: nodejs/web-servers/rjwebserver-example/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "rjwebserver-example",
+  "version": "1.0.0",
+  "description": "An example of a rjweb-server webserver running on Sparked Host",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Sparked Host (0x4096)",
```
Comment (hwalker928) [inline nodejs/web-servers/rjwebserver-example/package.json:9]: You are welcome to just put your username here.

File: nodejs/web-servers/rjwebserver-example/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  ctr.status(404)
+  ctr.print('This page was not found!')
+})
+
+routes.event('request', (ctr) => {
+  console.log(`${ctr.url.method} Request made to ${ctr.url.href} by ${ctr.client.ip}`)
+})
+
+const controller = webserver.initialize({
+  port: process.env.SERVER_PORT,
```
Comment (hwalker928) [inline nodejs/web-servers/rjwebserver-example/index.js:41]: I have a feeling that Pterodactyl sets this anyways without the need of a .env, but that would need checking

**Review by hwalker928 — APPROVED** [review]
Looks good to me! Thanks for your contribution. Would you also mind adding a README.md to the `rjwebserver-example` folder to describe what this does?

**Comment by rotvproHD** [comment]
Ill edit it a bit after I improved the webserver itself a little more

**Pull request merged**